codesigndocuitests/iostestrunner.go

```diff
@@ -0,0 +1,114 @@
+package codesigndocuitests
+
+import (
+	"fmt"
+	"path/filepath"
+	"strings"
+
+	"github.com/bitrise-io/go-utils/command"
+	"github.com/bitrise-io/go-utils/pathutil"
+	"github.com/bitrise-tools/go-xcode/plistutil"
+	"github.com/bitrise-tools/go-xcode/profileutil"
+)
+
+// IOSTestRunner ...
+type IOSTestRunner struct {
+	Path                string
+	InfoPlist           plistutil.PlistData
+	Entitlements        plistutil.PlistData
+	ProvisioningProfile profileutil.ProvisioningProfileInfoModel
+}
+
+// NewIOSTestRunners is the *-Runner.app which is generated with the xcodebuild build-for-testing command
+func NewIOSTestRunners(path string) ([]*IOSTestRunner, error) {
+	runnerPattern := filepath.Join(path, "*-Runner.app")
+	possibleTestRunnerPths, err := filepath.Glob(runnerPattern)
+	if err != nil {
+		return nil, err
+	}
+
+	if len(possibleTestRunnerPths) == 0 {
+		return nil, fmt.Errorf("no Test-Runner.app found in %s", path)
+	}
+
+	var testRunners []*IOSTestRunner
+	for _, testRunnerPath := range possibleTestRunnerPths {
+		infoPlist := plistutil.PlistData{}
```

Comment:
could not be used here this: https://github.com/bitrise-tools/go-xcode/blob/master/xcarchive/ios.go#L226

Reply:
It could, but I think we should not. They are similar but not 100%. I think it's better to have a different type for the Test-Runner.app

Reply:
👍